lnbits/wallets/lndrest.py

```diff
@@ -168,10 +183,9 @@ async def get_payment_status(self, checking_id: str) -> PaymentStatus:
         """
         # convert checking_id from hex to base64 and some LND magic
         try:
-            checking_id = base64.urlsafe_b64encode(bytes.fromhex(checking_id)).decode(
-                "ascii"
-            )
-        except ValueError:
+            checking_id = base64.b64encode(bytes.fromhex(checking_id)).decode("ascii")
+        except Exception as exc:
+            logger.error(f"LndRestWallet error get_payment_status: {str(exc)}")
```

Comment:
- is `str(exc)` required? If yes, then it should be present on line `170` also

Reply:
No, it is not needed.

See:

```
>>> ex = Exception("foo bar")
>>> str(ex)
'foo bar'
>>> f"{ex}"
'foo bar'
>>> f"{str(ex)}"
'foo bar'
```

Reply:
Yes, not needed:

```suggestion
            logger.error(f"LndRestWallet error get_payment_status: {exc}")
```